lib/notification.php

```diff
@@ -14,6 +14,15 @@ public static function sendMail($path){
 			return;
 		}
 		$email = \OCP\Config::getUserValue(\OCP\User::getUser(), 'settings', 'email', '');
+		if (\OCP\App::isEnabled(user_ldap)){
```

Comment:
We might even just display both user id and displayname - since both have a reasonable meaning